index.html

```diff
@@ -1,184 +1,288 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <link rel="preconnect" href="https://fonts.googleapis.com">
-    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
-    <link href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap" rel="stylesheet">
-    <link rel="stylesheet" href="style.css">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Poppins:ital,wght@0,300;0,400;0,500;0,600;0,700;0,800;1,200;1,300;1,400;1,500;1,600;1,700;1,800&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="style.css" />
     <title>Microverse Portfolio</title>
-</head>
-<body>
+  </head>
+  <body>
     <header>
-        <nav>
-            <a href="#">My Logo</a>
-            <img src="./assets/menu.png" alt="logo">
-        </nav>
-        <div class="background">
-            <div class="intro">
-                <div class="text">
-                    <h1>I'm Favour. <br> Glad to see you!</h1>
-                </div>
-                <p class="intro-text">I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hesitate to contact me.</p>
-                
-                <div class="socials">
-                    <h5>LET'S CONNECT</h5>
-                    <ul class="social-icon">
-                        <li> <a href="#"> <img src="./assets/twitter.svg" alt="twitter"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/Linkedin icon.svg" alt="Linkedin"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/medium.svg" alt="medium"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/github.svg" alt="github"> </a> </li>
-                        <li> <a href="#"> <img src="./assets/angellist.svg" alt="angellist"> </a> </li>
-                    </ul>
-                </div>
-            </div>
+      <nav>
```

Comment:
- [ ] Your `nav` still doesnt have `155px` padding on left and right side on bigger display. I tried to add some styling in chrome inspect dev tools. you can apply these stylings in starting. Paddings got corrected as I wanted but I had to disable `position: fixed`. you can apply these stylings and then you can try implementing position fixed for your nav.
your design before
![p](https://user-images.githubusercontent.com/96683843/236253696-6bb893e1-0637-406d-b2d4-1e75ca73b9c3.jpg)

styling I tried that gets padding fixed
![er](https://user-images.githubusercontent.com/96683843/236253726-abbaac93-cc58-4d67-b9a2-01cabd179511.jpg)

your design after
![f](https://user-images.githubusercontent.com/96683843/236253798-5909f7fa-4d91-49a0-8751-1d0a00ec57a7.jpg)